src/script/auth/page/ConversationJoin.tsx

```diff
@@ -219,62 +222,64 @@ const ConversationJoin = ({
             </Text>
           </ContainerXS>
         ) : renderTemporaryGuestAccountCreation ? (
-          <ContainerXS style={{margin: 'auto 0'}}>
-            <AppAlreadyOpen fullscreen={isPwaEnabled} />
-            <H2 style={{fontWeight: 500, marginBottom: '10px', marginTop: '0'}}>
-              <FormattedMessage
-                {...conversationJoinStrings.headline}
-                values={{
-                  brandName: Config.getConfig().BRAND_NAME,
-                }}
-              />
-            </H2>
-            <Text style={{fontSize: '16px', marginTop: '10px'}}>
-              <FormattedMessage {...conversationJoinStrings.subhead} />
-            </Text>
-            <Form style={{marginTop: 30}}>
-              <InputSubmitCombo>
-                <Input
-                  name="name"
-                  autoComplete="username"
-                  value={enteredName}
-                  ref={nameInput}
-                  onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
-                    resetErrors();
-                    setEnteredName(event.target.value);
+          <div>
```

Comment:
this is a fix for this bug https://stackoverflow.com/questions/54880669/react-domexception-failed-to-execute-removechild-on-node-the-node-to-be-re